README.md

```diff
@@ -168,6 +169,44 @@ require("obsidian").setup({
 })
 ```
 
+#### Templates support
+
+To insert a template, run the command `:ObsidianTemplate`. This will open [telescope.nvim](https://github.com/nvim-telescope/telescope.nvim) and allow you to select a template from the templates folder. Select a template and hit `<CR>` to insert. Substitution of `{{date}}`, `{{time}}`, and `{{title}}` is supported. 
+
+For example, with the following configuration
+
+```lua
+require("obsidian").setup({
+  dir = "~/my-vault",
+  templates = {
+      subdir = "my-templates-folder",
+      date_format = "%Y-%m-%d-%a",
+      time_format = "%H:%M"
+  }
+})
+```
+and the file `~/my-vault/my-templates-folder/note template.md`:
+
+```markdown
+---
+aliases = []
+---
+
+# {{title}}
+Date created: {{date}}
+```
+
+creating the note `Configuring Neovim.md` and executing `:ObsidianTemplate` will insert
+```markdown
+---
+aliases = []
+---
+
+# Configuring Neovim
+Date created: 2023-03-01-Wed
+```
+above the cursor position.
```

Comment:
I think it's more intuitive to insert *below* the cursor position. Is there a reason you chose to insert above?

Reply:
> Obsidian.md's default behaviour and it inserts the template at (before, if in using obsidian's vim mode in normal mode) the current cursor position, not linewise, which may be another option.

Actually I like that even better!

Reply:
I believe this works as in Obsidian.md now

Reply:
Nice!